test/stack_trace.spec.js

```diff
@@ -140,3 +139,81 @@ describe('getFilenameFromSourceMap', () => {
     expect(pathAndLine.line).to.be.equals(2)
   })
 })
+
+describe('getOriginalPathAndLine', () => {
+  const sourceMapsMap = new Map()
+  const setLRU = sinon.spy()
+  const getLRU = sinon.spy()
+  class LRU {
+    set (key, value) {
+      setLRU(key, value)
+      return sourceMapsMap.set(key, value)
+    }
+
+    get (key) {
+      getLRU(key)
+      return sourceMapsMap.get(key)
+    }
+  }
+
+  const { getOriginalPathAndLine } = proxyquire('../js/source-map', {
+    'lru-cache': LRU,
+    './node_source_map': nodeSourceMap,
+    fs: {
+      readFileSync: (filename) => {
+        switch (filename) {
+          case 'no-sourcemap':
+            return ''
+          case 'error':
+            throw new Error('errr')
+          default:
+            return readFileSync(filename).toString()
```

Comment:
why are you doing `.toString` with the result of the original `readFileSync` to create the mock of `readFileSync`? I think that this should be:
```suggestion
            return readFileSync(filename)
```